.github/workflows/build.yml

```diff
@@ -71,19 +71,29 @@ jobs:
         name: Checkout
         uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.base_ref }}
+          fetch-depth: 0
+      -
+        name: Prepare
+        id: prepare
+        run: |
+          echo ::set-output name=tag_name::${GITHUB_REF#refs/tags/}
+          echo ::set-output name=branch_name::$(git --no-pager branch -a --contains "$GITHUB_SHA" | grep 'remotes/origin/' | cut -d '/' -f3)
```

Comment:
This seems like it has the potential of choosing the wrong branch if a commit appears on multiple branches, such as we have just now on the `release/1_3` branch in this repo